RealmSwift/Migration.swift

```diff
@@ -80,7 +80,7 @@ extension Realm {
  instance provides access to the old and new database schemas, the objects in the Realm, and provides functionality for
  modifying the Realm during the migration.
  */
-public final class Migration {
+public struct Migration {
```

Comment:
Users can theoretically do stupid things like retroactively conforming `Migration` to a class-only protocol, but I don't think that's something we should worry about.